### PR TITLE
Add missing font for 3DTextPrefab

### PR DESCRIPTION
**Pull request by madvlad**
#377 

The `3DTextPrefab` came with a missing reference issue. Changed this to the originally intended Segoe UI font.

### CODE REVIEW

File: ProjectSettings/GraphicsSettings.asset [hunk 1]
```diff
@@ -40,7 +40,7 @@ GraphicsSettings:
     type: 0}
   m_TierSettings_Tier1:
     renderingPath: 1
-    useCascadedShadowMaps: 0
+    useCascadedShadowMaps: 1
```
Comment (S-Hodgson-MMOARgames) [inline ProjectSettings/GraphicsSettings.asset]: Please revert. Should be 0

**Comment by NeerajW** [comment]
@cre8ivepark @killerantz please chime in.

**Comment by killerantz** [comment]
The font (.ttf) cannot be included in HoloToolkit-Unity for licensing reasons. We do not have a license or rights to distribute this font on GitHub. The purpose of the prefab is to demonstrate how to have good crisp text with the intention that a developer will choose to use an imported font instead of the default Arial, which is included with Unity. Segoe UI is available on every Windows device so it should be easy enough to import. We considered using an open source font, but we did not find one with all the aspects of good typography we wanted to demonstrate as Segoe UI. I will defer to @cre8ivepark for any additional info.

**Comment by HodgsonSDAS** [comment]
I second just using Arial as default font.

**Comment by NeerajW** [comment]
@cre8ivepark what if @madvlad added the default Arial font to the prefab?

**Comment by cre8ivepark** [comment]
I need to check with font team but I don't think we can upload Arial font either. Even though it comes as default font in Windows, we cannot distribute font file online. That's the reason why we had to include instructions in this [readme.md file](https://github.com/Microsoft/HoloToolkit-Unity/tree/master/Assets/HoloToolkit/UI).
Thanks!

**Comment by HodgsonSDAS** [comment]
The Ariel font is already embedded into Unity. All we would have to do is assign the Ariel font in the inspector for the prefab and save it for the changes to take effect.

**Comment by cre8ivepark** [comment]
@HodgsonSDAS Another reason why developers have to import font file is that, we need texture of the font file that can be assigned to 3DTextSegoeUI that supports occlusion. But the Arial font in Unity does not have this texture. Step 1-3 in this Readme file - https://github.com/Microsoft/HoloToolkit-Unity/tree/master/Assets/HoloToolkit/UI

**Comment by HodgsonSDAS** [comment]
Sounds like we're going to need an open source font

**Comment by HodgsonSDAS** [comment]
https://fontlibrary.org/en/terms

**Comment by cre8ivepark** [comment]
I will update the files with open source alternative font - [https://github.com/Microsoft/Selawik](https://github.com/Microsoft/Selawik) with updated readme.md file. Thanks!

**Comment by cre8ivepark** [comment]
@HodgsonSDAS @NeerajW Submitted PR #498 It would be appreciated if you could check. Thank you!

**Pull request closed without merging**